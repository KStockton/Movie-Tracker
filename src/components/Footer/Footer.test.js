import React from 'react';
import Footer from './Footer';
import { shallow } from 'enzyme'

describe('Footer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow (<Footer/>)
  })

  it('Footer component should match the snapshot', () =>{
    expect(wrapper).toMatchSnapshot()
  })


})