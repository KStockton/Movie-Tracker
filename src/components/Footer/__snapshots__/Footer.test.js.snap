// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer component should match the snapshot 1`] = `undefined`;
